extensions/ql-vscode/src/query-history.ts

```diff
@@ -376,19 +406,23 @@ export class QueryHistoryManager extends DisposableObject {
       throw new Error(NO_QUERY_SELECTED);
     }
 
-    const textDocument = await vscode.workspace.openTextDocument(
-      vscode.Uri.file(finalSingleItem.query.program.queryPath)
+    if (!finalSingleItem.completedQuery) {
+      throw new Error('Select a completed query.');
+    }
```

Comment:
This may be annoying. Do we want to silently fail in this case?

Reply:
Yes, better to fail silently.

Reply:
I'll keep the explicit error in `findOtherQueryToCompare` since that may be confusing therwise.